components/clp-package-utils/clp_package_utils/scripts/compress.py

```diff
@@ -30,43 +30,40 @@
     validate_dataset_name,
 )
 
-logger = logging.getLogger(__file__)
+logger = logging.getLogger(__name__)
 
 
 def _generate_logs_list(
-    input_type: InputType,
     container_logs_list_path: pathlib.Path,
     parsed_args: argparse.Namespace,
 ) -> None:
-    if InputType.FS == input_type:
-        host_logs_list_path = parsed_args.path_list
-        with open(container_logs_list_path, "w") as container_logs_list_file:
-            if host_logs_list_path is not None:
-                with open(host_logs_list_path, "r") as host_logs_list_file:
-                    for line in host_logs_list_file:
-                        stripped_path_str = line.rstrip()
-                        if "" == stripped_path_str:
-                            # Skip empty paths
-                            continue
-                        resolved_path = pathlib.Path(stripped_path_str).resolve()
-                        mounted_path = CONTAINER_INPUT_LOGS_ROOT_DIR / resolved_path.relative_to(
-                            resolved_path.anchor
-                        )
-                        container_logs_list_file.write(f"{mounted_path}\n")
-
-            for path in parsed_args.paths:
-                resolved_path = pathlib.Path(path).resolve()
-                mounted_path = CONTAINER_INPUT_LOGS_ROOT_DIR / resolved_path.relative_to(
-                    resolved_path.anchor
-                )
-                container_logs_list_file.write(f"{mounted_path}\n")
-
-    elif InputType.S3 == input_type:
-        with open(container_logs_list_path, "w") as container_logs_list_file:
-            container_logs_list_file.write(f"{parsed_args.paths[0]}\n")
-
-    else:
-        raise ValueError(f"Unsupported input type: {input_type}.")
+    """
+    Generates logs list file for filesystem input.
+
+    :param container_logs_list_path: Path to write logs list
+    :param parsed_args: Parsed command-line arguments
```

Comment:
```suggestion
    :param container_logs_list_path: Path to write logs list.
    :param parsed_args: Parsed command-line arguments.
```
Pleiase don't forget adding period for complete docstring lines.